### PR TITLE
Mark DHP Verified

### PR DESCRIPTION
**Pull request by nsonanh**
### Upgrading Asset to Verified

If upgrading an Asset to Verified, please see the requirements specified at [LISTING](https://github.com/osmosis-labs/assetlists/blob/main/LISTING.md#upgrade-asset-to-verified-status-permissioned), and ensure the following:
- [x] Asset is defined thoroughly at the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - [x] A meaningful `description` and `extended_description`.
   - [x] Associated `socials`, including `website` and `twitter`.
   - [x] Logo Image has a square Aspect Ratio and < 250 KB file size.
- [x] This pool contains contains at least $1k USD-worth of liquidity of the asset (Provide Pool ID): **#1755**

'Verified' Status Validation Checklist (to be completed by Osmosis Zone maintainers):
- [x] Verify appearance and metadata
- [x] Accurate Price
- [x] Trading and routing functionality
- [x] Withdraw and Deposit
   - [x] Deposit Transaction URL (if in-app)

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Why are these items in the checklist ticked? When I looked, they are not true.
"A meaningful description and extended_description."
" Associated socials, including website and twitter."

In addition, upon deposit, the explorer URL was not functional. We need a functioning explorer.

**Comment by nsonanh** [comment]
> Why are these items in the checklist ticked? When I looked, they are not true. "A meaningful description and extended_description." " Associated socials, including website and twitter."

@JeremyParish69 These lines can't do ?
https://github.com/osmosis-labs/assetlists/blob/d4d54295228d62a08c996767bf7da3cd35b1d0ad/osmosis-1/osmosis-1.assetlist.json#L14968
https://github.com/osmosis-labs/assetlists/blob/d4d54295228d62a08c996767bf7da3cd35b1d0ad/osmosis-1/osmosis.zone_assets.json#L3503

> In addition, upon deposit, the explorer URL was not functional. We need a functioning explorer.
It's updated now, ptal

**Comment by JeremyParish69** [comment]
> > Why are these items in the checklist ticked? When I looked, they are not true. "A meaningful description and extended_description." " Associated socials, including website and twitter."
> 
> @JeremyParish69 These lines can't do ?
> 
> https://github.com/osmosis-labs/assetlists/blob/d4d54295228d62a08c996767bf7da3cd35b1d0ad/osmosis-1/osmosis-1.assetlist.json#L14968
> 
> 
> https://github.com/osmosis-labs/assetlists/blob/d4d54295228d62a08c996767bf7da3cd35b1d0ad/osmosis-1/osmosis.zone_assets.json#L3503
> 
> I looked at other projects and couldn't find the website anywhere
> 
> > In addition, upon deposit, the explorer URL was not functional. We need a functioning explorer.
> 
> It's updated now, ptal

Sorry, we can't update these files directly. I meant these details are missing at the Cosmos Chain Registry: https://github.com/cosmos/chain-registry/blob/master/dhealth/assetlist.json

File: osmosis-1/osmosis-1.chainlist.json [hunk 1]
```diff
@@ -7449,7 +7450,7 @@
       },
       "explorers": [
         {
-          "tx_page": "https://explorer.dhealth.com/dhealth/transactions/${txHash}"
+          "tx_page": "https://explorer.nodestake.org/dhealth/tx/${txHash}"
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.chainlist.json:7453]: The correction of the explorer must go in the zone_chains file

**Comment by nsonanh** [comment]
- Extended description & socials added: https://github.com/cosmos/chain-registry/pull/4641
- Explorer url updated: https://github.com/osmosis-labs/assetlists/pull/1582/commits/d786447f8010142c3beda593029a91f893b51bb9

PTAL

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -3502,8 +3502,10 @@
       "base_denom": "udhp",
       "twitter_URL": "https://twitter.com/dhealth_network",
       "path": "transfer/channel-75030/udhp",
-      "osmosis_verified": false,
-      "osmosis_unlisted": true,
+      "osmosis_verified": true,
+      "categories": [
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:3506]: can you please add a listing date?

File: osmosis-1/osmosis.zone_assets.json [hunk 3]
```diff
@@ -3502,8 +3502,10 @@
       "base_denom": "udhp",
       "twitter_URL": "https://twitter.com/dhealth_network",
       "path": "transfer/channel-75030/udhp",
-      "osmosis_verified": false,
-      "osmosis_unlisted": true,
+      "osmosis_verified": true,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:3505]: can you please set to false for now so we can validate with the updates?

File: osmosis-1/osmosis.zone_assets.json (hunk 2 above)
Comment (nsonanh) [reply osmosis-1/osmosis.zone_assets.json:3506]: @JeremyParish69 Does this have to be in the future ?

Comment (JeremyParish69) [reply osmosis-1/osmosis.zone_assets.json:3506]: It should be set to now, since it has the unlisted tag before

Comment (nsonanh) [reply osmosis-1/osmosis.zone_assets.json:3506]: `listing_date_time_utc` updated, ptal

**Comment by nsonanh** [comment]
@JeremyParish69 DHP is verified now but still we can't make swap:
image
Do you know why ? How can we fix it ?

**Comment by nsonanh** [comment]
https://sqsprod.osmosis.zone/router/quote?tokenIn=1000000ibc%2FFD506CCA1FC574F2A8175FB574C981E9F6351E194AA48AC219BD67FF934E2F33&tokenOutDenom=uosmo
returns 
```
{"message":"tokenInDenom is not a valid chain denom (ibc/FD506CCA1FC574F2A8175FB574C981E9F6351E194AA48AC219BD67FF934E2F33)"}
```

**Comment by JeremyParish69** [comment]
> @JeremyParish69 DHP is verified now but still we can't make swap: Do you know why ? How can we fix it ?

I wonder if it may have been a temporary issue. It works find for me on the Swap page from DHP to OSMO.

**Pull request merged**